### ISSUE TITLE
[a11y] [intermittent] Narrator does not focus Mozilla VPN

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#7959]
**Important details:**

* Not reproducing on Windows 11;
* Restarting the device usually fixes it;

**Mozilla VPN used versions:**

* 2.17.0 (2.202309082148);

**Platform tested:**

* Windows 10/11;

**Prerequisites:**

* Have a clean install of the Mozilla VPN;
* Have an account with subscription for Mozilla VPN;
* Have the Narrator enabled;

**Steps to reproduce:**

1. Focus the Mozilla VPN app by clicking on it (or ctrl+alt+tab );
1. Navigate inside the VPN using the keyboard;

**Expected results:**

* Mozilla VPN app is in focus. Narrator works inside of it while using the keyboard to navigate;

**Actual results:**

* Mozilla VPN app cannot be focused;
!11.09.2023_16.54.56_REC.mp4|width=480,height=800!

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-5498)
┆Reporter: Valentina Virlics

**Comment by data-sync-user** [comment]
➤ Vinoo Cherian commented:

Valentina Virlics Do you know if this works with NVDA (https://www.nvaccess.org/download/ ( https://www.nvaccess.org/download/|smart-link )) on Windows 10?

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Vinoo Cherian I tried with the NVDA screen reader and still does not work. Not sure if this type of details are visible in them, but I am attaching logs.

[^mozillavpn-2023-9-13.log]

**Comment by data-sync-user** [comment]
➤ Vinoo Cherian commented:

Since it repros with the NVDA and Narrator screen readers, this may be a Windows 10 bug that was fixed in Windows 11. (Doesn’t repro in Windows 11.)

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Currently we are not reproducing it anymore on our Wind 10 machines. When it happens again, we will also check other apps, and report back.

I can say for sure that the Narrator was working in apps like Skype and Thunderbird, while inside VPN was not.

Will give an update as soon as I reproduce the issue again. Thank you!

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Update: I was able to repro this, and while the Mozilla VPN client could not be focused, other apps content like: slack, outlook, chrome, could.

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

This is under investigation from QA while trying to provide some useful logs, by following Vinoo’s debugging instructions (provided via slack).

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

Marking as Blocked for the time being. Valentina Virlics can you move to “Backlog” or “To Do” when you are ready to pass the ball back to Eng please :)?

**Comment by data-sync-user** [comment]
➤ Vinoo Cherian commented:

Valentina did provide a debug log. Will reach out to QA if more is needed.

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Verified this while using the latest 2.21 Mozilla VPN build, on Windows 10.

The Mozilla VPN client is read by Narrator.
